src/main/java/io/appium/java_client/ios/IOSTouchAction.java

```diff
@@ -18,12 +34,15 @@ public IOSTouchAction(PerformsTouchActions performsTouchActions) {
      * @param el element to tap.
      * @param x  x offset.
      * @param y  y offset.
-     * @return this TouchAction, for chaining.
+     * @return this IOSTouchAction, for chaining.
+     * @deprecated use {@link #tap(ActionOptions)} with count=2 instead.
      */
+    @Deprecated
     public IOSTouchAction doubleTap(WebElement el, int x, int y) {
-        ActionParameter action = new ActionParameter("doubleTap", (HasIdentity) el);
-        action.addParameter("x", x);
-        action.addParameter("y", y);
+        ActionParameter action = new ActionParameter("doubleTap",
```

Comment:
Also
Won't the `doubleTap`  be supported anymore?

Reply:
It looks redundant to me. doubleTap can be easily replaced with tap(count:2) or double tap-wait-tap chains.
Would you like to keep it?